src/api/middlewares/auth.ts

```diff
@@ -0,0 +1,22 @@
+import { Request, Response, NextFunction } from 'express';
+import { paths } from '../../config/enviroments';
+import { UsersProxy } from '../users/UsersProxy';
+
+const userProxy = new UsersProxy(paths.configUsers());
+
+export default class AuthValidator {
+	checkJWT = async (
+		req: Request,
+		resp: Response,
+		next: NextFunction,
+	): Promise<Response | void> => {
+		const token: string = req.headers['authorization'];
+		console.log('---' + token + '---');
```

Comment:
Gente, perfeito, mas tirem esse log se não for estritamente necessário, para que eu possa aprovar